app/views/categories/show.html.erb

```diff
@@ -3,7 +3,7 @@
     <div>
```

Comment:
**[Optional]**

- [ ] In your application, I am not able to visit the expense pages :fearful: Kindly fix this issue :+1: